doc/user/data/m1_cluster_sizing.yml

```diff
@@ -0,0 +1,69 @@
+columns:
```

Comment:
Any changes to the M.1 sizes can be made here.  They'll be reflected in the create cluster and alter cluster pages.